src/esn.calendar.libs/app/components/event/form/event-form.pug

```diff
@@ -1,7 +1,7 @@
 //- NOTE: When adding form controls that should change the event in this form,
 //-       be sure to add them to the CAL_EVENT_MODIFY_COMPARE_KEYS constant, to
 //-       make sure they are checked during modification.
-form.event-form(role="form", name="form", aria-hidden="true", ng-class="{ 'readonly': !canModifyEvent}", ng-submit='submit()')
+form.event-form(role="form", name="form", aria-hidden="true", ng-class="{ 'readonly': !canModifyEvent}", ng-submit='submit()', ng-mouseover='dragModal()')
```

Comment:
I'm pretty convinced this is not the way it should b used. Can you add console.log() calls inside the dragModal() method, to see how many times .draggable() is called ?

Reply:
The `draggable` method should only be called once when the dialog first opens.

Reply:
OK, I'll have a look again. I try to do that in the first place but the .modal-dialog is loaded slower than the function, so jquery doesn't found it